### PR TITLE
Update BUILD since $(PLUGIN_OUT) is removed and $(OUT) placeholder replaced.

### PR DESCRIPTION
**Pull request by buildbreaker2021**

### CODE REVIEW

**Comment by meteorcloudy** [comment]
Thanks! Can you look into the presubmit failure?

**Comment by shahms** [comment]
This file is part of our release; in which released version of Bazel is this change live?

**Comment by meteorcloudy** [comment]
https://github.com/bazelbuild/bazel/commit/d69a55cf7b64df1f8a86a31e35a8a0b10cabeabc is only available at HEAD right now, it's not cut into Bazel 5.0. So probably in later rolling releases and 6.0.

**Comment by shahms** [comment]
I don't see how I can merge this, then. While building Kythe at Bazel HEAD will fail without this change, doing so with an older version will silently do the wrong thing.  While getting rid of `$(PLUGIN_OUT)` seems feasible, changing `$(OUT)` to "%s" is a breaking change with no available workaround.

File: kythe/extractors/BUILD [hunk 1]
```diff
@@ -51,7 +51,7 @@ proto_lang_toolchain(
         "@com_google_protobuf//:type_proto",
         "@com_google_protobuf//:wrappers_proto",
     ],
-    command_line = "--$(PLUGIN_OUT)=:$(OUT)",
+    command_line = "--PLUGIN_java_stubby_out=:%s",
```
Comment (shahms) [inline kythe/extractors/BUILD:54]: AFAIK, this should be `--PLUGIN_C++_out` (although actually *determining* that, without the aid of the `$(PLUGIN_OUT)` replacement) is incredibly challenging.  I had to run `bazel -s` to do so.

**Comment by meteorcloudy** [comment]
@buildbreaker2021 Is there any easy way to migrate this in a way that works with both newer and older Bazel? Otherwise, I think we probably should rollback bazelbuild/bazel@d69a55c first, and reintroduce it in a way that's easier to migrate for.

**Comment by buildbreaker2021** [comment]
Will try something first.

**Comment by shahms** [comment]
I believe this works, thanks!

**Review by shahms — APPROVED** [review]

**Pull request merged**